config/src/main/java/com/quorum/tessera/config/adapters/PathAdapter.java

```diff
@@ -11,12 +12,17 @@
 
     @Override
     public Path unmarshal(final String input) {
-        return (input == null) ? null : Paths.get(input);
+
+        return Optional.ofNullable(input)
```

Comment:
Was this flagged by the plugin? Not sure this is any better than what we had before.